pages/checkoutPage.tsx

```diff
@@ -104,6 +108,61 @@ const CheckoutPage = () => {
         )
     }
 
+    useEffect(() => {
+        axios
+            .post(`${paymentapiUrl}/get-token`, {
+                customerName: 'OSC',
+                widgetID: '393ed315-c1a0-4978-bc92-fbc55793b180',
+            })
+            .then((response) => {
+                setToken(response.data.data.access_token)
+            })
+            .catch((error) => {
+                console.error('Error fetching token:', error)
+            })
+    }, [])
+
+    useEffect(() => {
+        if (initRequest.data) {
+            const amount =
+                initRequest.data[0].message.catalogs.responses[0].message.order
```

Comment:
please abstract some of the function to a utility file and use those in this page to avoid bloating in the component

Reply:
done @ujjwal502